src/functions-and-arrays.js

```diff
@@ -1,34 +1,71 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(numb1, numb2) {
+  if (numb1 < numb2) {
+    return numb2;
+} else if (numb1 > numb2) {
+    return numb1;
+  
+  } 
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(arr) {
+  let longestWord = '';
+  for (let i = 0; i < arr.length; i++) {
+    if (arr.length === 0) {
+      return null;
+    } else if (arr[i].length > longestWord.length) {
+      longestWord = arr[i];
+    }
+  }
+  return longestWord;
+}
 
```

Comment:
here it's just a small thing, the condition to return null needs to be outside of the for loop:
```
function findLongestWord(arr) {
  let longestWord = '';
  if (arr.length === 0) {
    return null;
  }
  for (let i = 0; i < arr.length; i++) {
    if (arr[i].length > longestWord.length) {
      longestWord = arr[i];
    }
  }
  return longestWord;
}

```